src/backend/main/py.main/caliopen_main/interfaces/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .IO import *
```

Comment:
no import *, be declarative of what to be imported from a package or use `import <package>` syntax